### ISSUE TITLE
Outdated JDA?

### ISSUE THREAD

**Issue by PulzHF** [Eufranio/MagiBridge#131]
I haven't changed my MagiBridge (7-2.5) & DiscordLink (7-1.5) versions in forever- as they functioned fine. I have also not adjusted the configs in almost as long.

The issue started exactly when I activated my commercial license on Discord.

https://pastebin.com/TW417EsC

The above occurs at startup immediately after 
```
[net.dv8tion.jda.core.JDA]: Login Successful!
[net.dv8tion.jda.core.requests.WebSocketClient]: Connected to WebSocket
```

MBDiscordLink then starts throwing a fit every 30 seconds or so with the following:
https://pastebin.com/SYeDKN9x

**Comment by Asherslab** [comment]
Could you please define what you mean by "Activated my commercial license"
Are you a game dev? if so i may actually know the issue

**Comment by PulzHF** [comment]
> Could you please define what you mean by "Activated my commercial license"
> Are you a game dev? if so i may actually know the issue.

The Developer Application License, which grants you access to more server features. Like I said, both of my bots stopped working immediately after enabling my license.

**Comment by PulzHF** [comment]
I have noticed and reported an issue to Discord regarding the Dev license seemingly stopping the generation of new Webhooks. Is this related, perchance?

**Comment by Asherslab** [comment]
no, and the issue if i'm not mistakes is that the version of MagiBridge that you're most likely using, is most likely using a version of JDA that does not support the new types of channels that discord has added (for things like the store and etc). my first suggestion is to update magibridge (make a backup first)
if that doesn't work @Eufranio may need to update the JDA version

**Comment by PulzHF** [comment]
Can confirm the issue is still present on the latest MB.

**Comment by Asherslab** [comment]
okay. @Eufranio i think you need to update the JDA version =D

**Comment by PulzHF** [comment]
Don't suppose there's any news on this?

**Comment by PulzHF** [comment]
The Classic Pack will be using Discord Integration in the meantime. I'd appreciate a mention on Discord if/when this is fixed, cheers.

**Comment by Asherslab** [comment]
@Eufranio ?

**Comment by Eufranio** [comment]
I'm rewritting a good part of MagiBridge, I've already updated JDA on my dev env, I'll let you know when it's done.

**Comment by Pulz-TCP** [comment]
Fixed with 2.7 rewrite, many thanks.
